src/applications/personalization/profile/components/proof-of-veteran-status/ProofOfVeteranStatusNew.jsx

```diff
@@ -389,11 +389,22 @@ const ProofOfVeteranStatusNew = ({
 
             {useLighthouseApi ? (
               <>
```

Comment:
It feels like this should be more of a case statement like (pseudo-code):
```
case
when errors.length > 0 
  ...
when 'confirmed'
 ...
 when 'not confirmed'
  ...
else
  null
```

TBH, I've always found the widespread use of multi-line ternaries in jsx really hard to read. But people seem to do it everywhere in react so 🤷